$OpenBSD: patch-src_main_c,v 1.2 2009/10/31 12:26:25 edd Exp $

set some sane defaults:

use mplayer's software volume control (see pkg/MESSAGE)
disable bar animation in full screen

--- src/main.c.orig	Thu Sep 17 17:56:06 2009
+++ src/main.c	Sat Oct 17 17:53:22 2009
@@ -699,7 +699,7 @@ int main(int argc, char *argv[])
     playlist_visible = FALSE;
     disable_fullscreen = FALSE;
     disable_framedrop = FALSE;
-    softvol = FALSE;
+    softvol = TRUE;
     subtitlefont = NULL;
     subtitle_codepage = NULL;
     subtitle_color = NULL;
@@ -711,7 +711,7 @@ int main(int argc, char *argv[])
     reallyverbose = 0;
     embedding_disabled = FALSE;
     disable_pause_on_click = FALSE;
-    disable_animation = FALSE;
+    disable_animation = TRUE;
     auto_hide_timeout = 3;
     use_mediakeys = TRUE;
     use_defaultpl = FALSE;
